index.html

```diff
@@ -15,5 +15,87 @@
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
+<header>
```

Comment:
Notice how all the elements inside `head` (lines 4-10) are uninformly indented. 
You should do the same here so that it's clear that `header` is inside `body`